internal/pkg/task/env_runner.go

```diff
@@ -53,16 +54,16 @@ func (r *EnvRunner) Run() ([]*Task, error) {
 		return nil, fmt.Errorf("get cluster for environment %s: %w", r.Env, err)
 	}
 
-	filters := r.filtersForVPCFromAppEnv()
-
-	subnets, err := r.VPCGetter.PublicSubnetIDs(filters...)
+	description, err := r.EnvironmentDescriber.Describe()
 	if err != nil {
 		return nil, fmt.Errorf(fmtErrPublicSubnetsFromEnv, r.Env, err)
```

Comment:
Can we have a `const fmtErrDescribeEnvironment = "describe environment %s: %w"` and return the error as follows 🙏 :

```suggestion
		return nil, fmt.Errorf(fmtErrDescribeEnvironment, r.Env, err)
```